fbw-a380x/src/wasm/extra-backend-a380x/src/AircraftPresets/AircraftPresetProcedures_A380X.h

```diff
@@ -165,7 +165,7 @@ class AircraftPresetProcedures_A380X {
         ProcedureStep{"Transponder On",       2080, false, 1000,  "(L:A32NX_TRANSPONDER_MODE) 1 ==",                  "1 (>L:A32NX_TRANSPONDER_MODE)"},
         ProcedureStep{"ATC ALT RPTG On",      2090, false, 1000,  "(L:A32NX_SWITCH_ATC_ALT) 1 ==",                    "1 (>L:A32NX_SWITCH_ATC_ALT)"},
         ProcedureStep{"TCAS TRAFFIC ABV",     2100, false, 2000,  "(L:A32NX_SWITCH_TCAS_TRAFFIC_POSITION) 2 ==",      "2 (>L:A32NX_SWITCH_TCAS_TRAFFIC_POSITION)"},
-        ProcedureStep{"Autobrake Max",        3080, false, 2000,  "(L:A32NX_AUTOBRAKES_ARMED_MODE) 3 ==",             "3 (>L:A32NX_AUTOBRAKES_ARMED_MODE_SET)"},
+        ProcedureStep{"Autobrake RTO",        3080, false, 2000,  "(L:A32NX_AUTOBRAKES_ARMED_MODE) 6 ==",             "1 (>L:A32NX_OVHD_AUTOBRK_RTO_ARM_IS_PRESSED)"},
```

Comment:
This is not correct in how you setup the the vars - you would basically hold the button all the time (it is not a switch). See remark above. 

Correct with your current button logic would be:
```cpp
        ProcedureStep{"Autobrake RTO",        3068, false,  200,  "(L:A32NX_AUTOBRAKES_RTO_ARMED) 1 ==",                "1 (>L:A32NX_OVHD_AUTOBRK_RTO_ARM_IS_PRESSED)"},
        ProcedureStep{"Autobrake RTO",        3068, false, 2000,  "(L:A32NX_OVHD_AUTOBRK_RTO_ARM_IS_PRESSED) 0 ==",     "0 (>L:A32NX_OVHD_AUTOBRK_RTO_ARM_IS_PRESSED)"},
```
Not very nice as one has to manually reset the "pressed" push button to 0:

Reply:
see above - resolved